src/machines/java/com/enderio/machines/common/blockentity/SoulBinderBlockEntity.java

```diff
@@ -35,40 +35,62 @@
 import org.jetbrains.annotations.Nullable;
 
 import java.util.List;
+import java.util.Objects;
 
 import static com.enderio.base.common.util.ExperienceUtil.EXPTOFLUID;
 
-public class SoulBinderBlockEntity extends PoweredCraftingMachine<SoulBindingRecipe, SoulBindingRecipe.Container> {
+public class SoulBinderBlockEntity extends PoweredMachineEntity {
```

Comment:
This is the most important read; this is what a converted machine will look like.